File: marvelHeroes/src/main/java/com/futureworkshops/marvelheroes/domain/rx/scheduler/SchedulersProvider.java
/*
 * Copyright (c) 2018 FutureWorkshops. All rights reserved.
 */

package com.futureworkshops.marvelheroes.domain.rx.scheduler;

import io.reactivex.Scheduler;

public interface SchedulersProvider {

    Scheduler io();

    Scheduler ui();
}

File: marvelheroeskotlin/src/main/java/com/futureworkshops/marvelheroeskotlin/data/rx/SchedulersProvider.kt
/*
 * Copyright (c) 2018 FutureWorkshops. All rights reserved.
 */

package com.futureworkshops.marvelheroeskotlin.data.rx

import io.reactivex.Scheduler

/**
 * Created by dimitrios on 20/06/2018.
 * Copyright © 2018 - Future Workshops.
 * All rights reserved.
 */

interface SchedulersProvider {
    
    fun io(): Scheduler
    
    fun ui(): Scheduler
    
}